cherab/jet/machine/cad_files.py

```diff
@@ -673,9 +673,21 @@
 
 
 def import_jet_mesh(world, override_material=None, tungsten_material=None, beryllium_material=None,
-                    lambert_material=None, verbose=True):
-
-    for mesh_item in JET_MESH:
+                    lambert_material=None, verbose=True, mesh_description=JET_MESH):
```

Comment:
I don't think `mesh_description` is the best name for this argument. How about `mesh_components` instead?

Reply:
I'm not sure here. You have to pas a list of tuples containing path to the file and material. That is closer to description than components. What about component_description?

Reply:
I certainly think you want to pluralise whatever you choose, so that it's clear that this is a list of things you want to load. The docstring can describe what form is necessary for each element in `components`, but it should be reasonably clear to anybody reading code calling this function what is going on. And to me something like 
```python
component_description = VACUUM_VESSEL + DIAGNOSTICS + DIVERTOR_TILES
```
makes less sense than
```python
mesh_components = VACCUM_VESSEL + DIAGNOSTICS + DIVERTOR_TILES
```
The latter just reads more naturally, at least to me as a native English speaker.

Reply:
Changed to mesh_components